cmd/contour/ingressstatus.go

```diff
@@ -62,62 +65,76 @@ func (isw *loadBalancerStatusWriter) Start(stop <-chan struct{}) error {
 		isw.log.Info("elected leader")
 	}
 
-	var shutdown chan struct{}
-	var ingressInformers sync.WaitGroup
+	u := &k8s.StatusAddressUpdater{
+		Logger: func() logrus.FieldLogger {
+			// Configure the StatusAddressUpdater logger.
+			log := isw.log.WithField("context", "StatusAddressUpdater")
+			if isw.ingressClass != "" {
+				return log.WithField("target-ingress-class", isw.ingressClass)
+			}
+
+			return log
+		}(),
+		IngressClass:  isw.ingressClass,
+		StatusUpdater: isw.statusUpdater,
+		Converter:     isw.Converter,
+	}
+
+	// Create new informer for the new LoadBalancerStatus. The
```

Comment:
nit: this comment is slightly stale, could be clarified since we're no longer creating a new informer every time the LB address changes.

Reply:
The comment doesn't say that it is creating a new informer every time?

Reply:
it says "for the new LoadBalancerStatus", which came from the old code where when a new LB address was received, a new informer would be created. Can we just say "Create informers" or something similar that doesn't reference the old code structure?

Reply:
Ah I see ... fixed.